### PR TITLE
Don't modify the array passed into array_to_img

### PR DESCRIPTION
**Pull request by danieljl**
Scale image array to display, but not the original one.

### CODE REVIEW

**Comment by fchollet** [comment]
Send this PR to the `keras-2` branch instead.

**Comment by danieljl** [comment]
@fchollet Done.

File: keras/preprocessing/image.py [hunk 1]
```diff
@@ -253,8 +253,8 @@ def array_to_img(x, data_format=None, scale=True):
         x = x + max(-np.min(x), 0)
```
Comment (fchollet) [inline keras/preprocessing/image.py]: This line already creates a copy of the array. Therefore subsequent lines do not modify the original array in place, they modify the copy.

Comment (danieljl) [inline keras/preprocessing/image.py]: Ah yes it's been fixed at #5070.

**Pull request closed without merging**